model/DataQuery.php

```diff
@@ -779,6 +779,22 @@ public function selectFromTable($table, $fields) {
 	}
 
 	/**
+	 * Add the given fields from the given table to the select statement.
+	 *
+	 * @param String $table Unquoted table name (will be escaped automatically)
+	 * @param Array $fields Database column names (will be escaped automatically)
+	 */
+	public function addSelectFromTable($table, $fields) {
+		$fieldExpressions = array_map(function($item) use($table) {
+			return "\"$table\".\"$item\"";
```

Comment:
At the moment symbol2sql seems hard coded to ansi-quotes, so it's probably safe for the time being. I wish I never added this method though; It's made the ORM api a lot more inconsistent.